.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "example/ios/Kaluga-SwiftUI"]
```

Comment:
So we reuse it in multiple projects as git submodule, what if convert it into Swift Package Manager, so it can be easily integrated into project

Reply:
Its generated Swift code through Sourcery and requires a config file. Afaik thats not really possible with SPM?

Reply:
After a bit of Googling I guess it might be possible... looking into it a bit more for sure, cause in that case its a good suggestion